Userland/Utilities/sync.cpp

```diff
@@ -1,12 +1,14 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2022, Alex Major
```

Comment:
Please add an email to this.

Reply:
I would rather not have my email in the source. I use a noreply GitHub email in these commits and the contributing guide or linter does not enforce an email.